package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@google-cloud/monitoring-dashboards",
+  "version": "0.1.0",
+  "description": "Dashboard client for Node.js",
```

Comment:
@alexander-fenster is this file coming from the generator?  I would expect a better description by default

Reply:
I pulled a better description from the product page.

Reply:
Thanks :) Could I trouble you to open an upstream issue on the generator so we don't have to do this every time?